R/state.R

```diff
@@ -0,0 +1,22 @@
+check_pre_release_state = function(which, force = FALSE) {
+
+  # check version number
+  if (length(strsplit(as.character(desc::desc_get_version()), "[.]|-")[[1]]) < 4) {
+    ui_oops("pre_release(): Package versions needs to indicate a dev version before calling pre_release().")
```

Comment:
Can we use `ui_code()` inside `ui_oops()` ?

Reply:
Done. What about #38 ?